zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeer.java

```diff
@@ -2278,7 +2278,6 @@ public QuorumCnxManager createCnxnManager() {
                 this.getView(),
                 this.authServer,
                 this.authLearner,
-                this.tickTime * this.syncLimit,
```

Comment:
This seems unrelated

Reply:
This corresponds to removing `int socketTimeout` from the constructor of QuorumCnxManager.

Reply:
Sorry I misread the diff